config/locales/en/activities.yml

```diff
@@ -0,0 +1,6 @@
+en:
+  activity_mailer:
+    layout:
```

Comment:
the name of the key here is not random, you must follow the i18n convention.
REF: http://guides.rubyonrails.org/i18n.html#translations-for-action-mailer-e-mail-subjects

So that you can use short hand key like `t('.greeting')` in your views